UnrealPlugin/README.md

```diff
@@ -47,4 +123,52 @@ public class <projectname>ServerTarget : TargetRules
 	}
 }
 
-```
\ No newline at end of file
+```
+
+Update DefaultGame.ini to show the following:
+ [/Script/UnrealEd.ProjectPackagingSettings]
+IncludeAppLocalPrerequesites=True
+
+If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
+
+Update DefaultEngine.ini file and add this:
+[/Script/EngineSettings.GameMapsSettings]
+GameInstanceClass=/Script/[game name].MyGameInstance"
```

Comment:
This can also be set through the UI in the editor. In the editor go to Edit -> Project Settings. In the opened window navigate to Maps&Modes on the left side. Scroll to the bottom, and then you can set the option "Game Instance Class" to your new game instance class directly, so avoid typos.